lib/jekyll-github-metadata/value.rb

```diff
@@ -39,14 +42,6 @@ def render
         raise e
       end
 
-      def to_s
-        render.to_s
-      end
-
-      def to_json(*)
```

Comment:
Unfortunately, we can't just alias this method to `#render`. `to_json` must be able to accept a `state` parameter. We [do that in Jekyll::Drops::Drop](https://github.com/jekyll/jekyll/blob/master/lib/jekyll/drops/drop.rb#L153-L161) to great effect. (See how `state` affects things in [`DocumentDrop#hash_for_json`](https://github.com/jekyll/jekyll/blob/master/lib/jekyll/drops/document_drop.rb#L40-L53).)

```ruby
def to_json(state = nil)
  require "json"
  JSON.generate(value, state)
end
```